lib/tmuxinator/config.rb

```diff
@@ -70,6 +82,8 @@ def validate(name, options={})
 
         config_path = Tmuxinator::Config.project(name)
 
+        settings = parse_settings(args)
```

Comment:
That's what you think. This variable can actually be accessed from within the YAML configs!

*I'm arguing with a robot CI dog*